python/ml4ir/base/model/architectures/dnn.py

```diff
@@ -28,7 +28,10 @@ def __init__(self, model_config: dict, feature_config: FeatureConfig, file_io):
         self.layer_ops: List = self.define_architecture(model_config, feature_config)
         if 'positional_bias_handler' in self.model_config and self.model_config['positional_bias_handler'][
             'key'] == PositionalBiasHandler.FIXED_ADDITIVE_POSITIONAL_BIAS:
-            self.positional_bias_layer = FixedAdditivePositionalBias(max_ranks=self.model_config['positional_bias_handler']['max_ranks'])
+            self.positional_bias_layer = FixedAdditivePositionalBias(max_ranks=self.model_config['positional_bias_handler']['max_ranks'],
```

Comment:
@mohazahran Awesome! Can you update the tests as well?

Reply:
The positional bias test focuses on the conversion to one-hot in training and testing. These changes here would not interfere with the current test.

Reply:
I would recommend checking the config for that layer and also check if the initialization is all 0s.

Reply:
ok will add, extra checks